### PR TITLE
Create LoadBalancer instance even if no Nodes are available

### PR DESCRIPTION
**Pull request by Ashughorla**
# Proposed Changes

- Change condition of creating LB instance 
- Test case

Fixes https://github.com/onmetal/cloud-provider-onmetal/issues/224

### CODE REVIEW

File: pkg/cloudprovider/onmetal/load_balancer_test.go [hunk 1]
```diff
@@ -131,9 +150,8 @@ var _ = Describe("LoadBalancer", func() {
 	It("should ensure external LoadBalancer", func() {
 
 		By("expecting an error if there are no nodes available")
```
Comment (afritzler) [inline pkg/cloudprovider/onmetal/load_balancer_test.go:152]: Can you please adjust the description here

**Review by afritzler — CHANGES_REQUESTED** [review]

File: pkg/cloudprovider/onmetal/load_balancer_test.go (hunk 1 above)
Comment (Ashughorla) [reply pkg/cloudprovider/onmetal/load_balancer_test.go]: Yes sure. Addressed.

**Comment by afritzler** [comment]
I am actually not sure if we really need the check for the `ClusterIP` type as this is checked here: https://github.com/kubernetes/cloud-provider/blob/master/controllers/service/controller.go#L369

**Comment by Ashughorla** [comment]
@afritzler Updated the changes. Please have a look.

**Review by afritzler — APPROVED** [review]

**Review by afritzler — APPROVED** [review]

**Pull request merged**